src/inmanta/server/services/resourceservice.py

```diff
@@ -748,7 +747,7 @@ async def resource_action_update(
         started: datetime.datetime,
         finished: datetime.datetime,
         status: Optional[Union[const.ResourceState, const.DeprecatedResourceState]],
-        messages: List[Dict[str, Any]],
```

Comment:
This I believe is not possible, we can't just change this part of the external api?